README.md

```diff
@@ -2,3 +2,119 @@
 
 
 ## interfaces
+
+## Quick Example of adding a `MessageOption`
```

Comment:
this doesn't seme like a long term place to put this kind of content imo

Reply:
I'll make a doc instead

Reply:
remove the extra content https://github.com/ductone/protoc-gen-pgdb/pull/63/commits/2ab135ac0f5877c52c6e1be474ae75d787adf195